### PR TITLE
Android: Gradle 4, Android plugin 3, SDK 27

### PR DESCRIPTION
**Pull request by devemux86**
Android has released new major versions in all tools, so we should advance the sample too.

~But first the PR demonstrates immediately the current failed use of GraphHopper on Android, as described in #1157, if not commenting the mentioned code.~
EDIT: that should be fixed via #1207.

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -1,15 +1,17 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     repositories {
+        google()
```
Comment (karussell) [inline android/build.gradle]: Is this necessary :) ?

Comment (devemux86) [reply android/build.gradle]: `google()` is a now the formal repository for Android related libraries, like plugins, support libs, etc. ([here](https://dl.google.com/dl/android/maven2/index.html) are the contents).

**Comment by devemux86** [comment]
> BTW: Does it work with the mmap fix now?

Yes I tested it, all seem working ok again!

**Comment by karussell** [comment]
Cool, thanks!

**Comment by karussell** [comment]
One last comment: would you mind signing the commit? (if not possible yet, no problem & no hurry, we can do this later then)

**Comment by devemux86** [comment]
Signed and force pushed, should be ok now?

**Comment by karussell** [comment]
Cool, thanks!

**Pull request merged**